config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+  get 'errors/file_not_found'
+
+  get 'errors/unprocessable'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.